extensions/controllers/sandboxclaim_controller.go

```diff
@@ -38,6 +40,10 @@ import (
 	extensionsv1alpha1 "sigs.k8s.io/agent-sandbox/extensions/api/v1alpha1"
 )
 
+const (
+	sandboxLabel = "agents.x-k8s.io/sandbox-name-hash"
```

Comment:
It's is defined in both controllers and extensions/controllers. Since this label represents a contract between the two controllers, defining it in two places risks consistency drift. Consider reusing it.

Reply:
That's true, I'm more than happy to tackle that in a separate PR #216 (i've created an issue on this) since this could modify sandbox core files (controller and tests).